Source/DafnyCore/Resolver/PreType/PreTypeResolve.ActualParameters.cs

```diff
@@ -127,7 +127,7 @@ internal void ResolveActualParameters(ActualBindings bindings, List<Formal> form
             whatKind + (context is Method ? " in-parameter" : " parameter"));
 
           Constraints.AddSubtypeConstraint(
-            formal.PreType.Substitute(typeMap), b.Actual.PreType, callTok,
+            formal.PreType.Substitute(typeMap), b.Actual.PreType, callTok /* TODO should be b.Actual.Origin */,
```

Comment:
Same question here (but only reply in one place and mark this as resolved)

Reply:
When using `callTok`, if there is a problem with one of the arguments, then the error is reported on the call `(` instead of on the center of the argument.